qvisor/src/vmspace/nvidia.rs

```diff
@@ -135,6 +136,30 @@ pub fn NvidiaProxy(cmd: ProxyCommand, parameters: &ProxyParameters) -> Result<i6
             // );
             return Ok(ret as i64);
         }
+        ProxyCommand::CudaUnRegisterFatBinary => {
+            let moduleKey = parameters.para1;
+            //error!("unregister module key:{:x}", moduleKey);
+
+            let mut module = match MODULES.lock().get(&moduleKey){
+                Some(module) => {
+                    //error!("module: {:x} for this module key: {:x} has been found",module, moduleKey);
+                    *module 
+                }
+                None => {
+                    //error!("no module be found with this fatcubinHandle:{:x}",moduleKey);
```

Comment:
Actually we do need to error log here because here is real error. Please uncomment. Please search for other "None => " locations and uncomments error log.